spec/lexer_spec.cr

```diff
@@ -137,14 +137,14 @@ describe Lexer do
   it_lexes_string %('''\nhello  \n  world'''), "hello  \n  world"
   it_lexes_string %('''I don't need'''), "I don't need"
 
-  it_lexes_time "1979-05-27T07:32:00Z", Time.new(1979, 5, 27, 7, 32, 0, kind: Time::Kind::Utc)
-  it_lexes_time "1979-05-27T07:32:00-07:30", Time.new(1979, 5, 27, 15, 2, 0, kind: Time::Kind::Utc)
-  it_lexes_time "1979-05-27T07:32:00+07:30", Time.new(1979, 5, 27, 0, 2, 0, kind: Time::Kind::Utc)
+  it_lexes_time "1979-05-27T07:32:00Z", Time.utc(1979, 5, 27, 7, 32, 0)
+  it_lexes_time "1979-05-27T07:32:00-07:30", Time.utc(1979, 5, 27, 15, 2, 0)
+  it_lexes_time "1979-05-27T07:32:00+07:30", Time.utc(1979, 5, 27, 0, 2, 0)
   it_lexes_time "1979-05-27T07:32:00.999999-07:00",
     {% if Crystal::VERSION =~ /^0\.(\d|1\d|2[0-3])\./ %}
-       Time.new(1979, 5, 27, 14, 32, 0, 999, kind: Time::Kind::Utc)
+       Time.utc(1979, 5, 27, 14, 32, 0, 999)
     {% else %}
-       Time.new(1979, 5, 27, 14, 32, 0, nanosecond: 999999000, kind: Time::Kind::Utc)
+       Time.utc(1979, 5, 27, 14, 32, 0, nanosecond: 999999000)
     {% end %}
```

Comment:
Not sure to still keep this ugly macros, especially we can't ensure the code will work on older Crystal compilers.